samples/ConsoleApp/Program.cs

```diff
@@ -34,12 +39,47 @@ private static async Task RunAsync()
                 Console.WriteLine("Calling regularHttpClient.GetAsync");
                 await regularHttpClient.GetAsync("default-handler");
                 Console.WriteLine("Called regularHttpClient.GetAsync");
-
-                var client = new HttpClient();
+                
                 Console.WriteLine("Calling client.GetAsync");
                 await client.GetAsync("http://127.0.0.1:8080/api/mongo");
                 Console.WriteLine("Called client.GetAsync");
             }
+            
+            var request = new HttpRequestMessage { RequestUri = new Uri("https://www.example.com/"), Method = HttpMethod.Post, Content = new StringContent(string.Empty, Encoding.UTF8), };
+
+            var tracer = GlobalTracer.Instance;
+            using (var scope = tracer.BuildSpan("Client POST " + request.RequestUri)
+                                     .WithTag("span.kind", "client")
+                                     .StartActive())
+            {
+                var contextPropagationHeaders = new Dictionary<string, string>();
+                var contextCarrier = new TextMapInjectAdapter(contextPropagationHeaders);
+                tracer.Inject(scope.Span.Context, BuiltinFormats.TextMap, contextCarrier);
+                foreach (var kvp in contextPropagationHeaders)
+                {
+                    request.Headers.Add(kvp.Key, kvp.Value);
+                }
+
+                using var response = await client.SendAsync(request);
+
+                scope.Span.SetTag("http.status_code", (int)response.StatusCode);
+
+                var responseContent = await response.Content.ReadAsStringAsync();
+                scope.Span.SetTag("response.content", responseContent);
+                scope.Span.SetTag("response.length", responseContent.Length);
+
+                foreach (var header in response.Headers)
```

Comment:
For our purposes here the two tags above are enough: we don't need the other tags added in this foreach, they make nice for a sample but not really adding value here.